spec/features/featured_item_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'spec_helper'
+
+RSpec.describe "featuring items" do
```

Comment:
We usually just put `describe` instead of `RSpec.describe` although I'm not sure it matters.